neqo-http3/src/connection.rs

```diff
@@ -47,8 +47,8 @@ pub enum Http3State {
     ZeroRtt,
     Connected,
     GoingAway(u64),
-    Closing(CloseError),
-    Closed(CloseError),
+    Closing(ConnectionError),
+    Closed(ConnectionError),
```

Comment:
I do not know why this was not using ConnectionError. It should have been.

Reply:
Easy mistake to make :)